### ISSUE TITLE
Easy multicore with neural networks using synaptic?

### ISSUE THREAD

**Issue by bootsie123** [cazala/synaptic#320]
I've created a neural network with synaptic with 784 input layers, 100 hidden layers, and 10 output layers. What my neural network is doing is taking in an image of a number from 0-9 and then tries to figure out which number it actually is. I'm trying to cut down on the time it takes to train. Currently using 20 iterations it takes 7 minutes. This is in no way ideal.

I've been looking into seeing if I could use multiple cores using cluster or something like it. However, I think that's pretty much a dead end with how synaptic works if I understand it correctly. Since if you run multiple trainings at the same time they wouldn't combine together for the finished product. I may be wrong though. So please correct me if I am.

**Conclusion:**

So basically, what I was wondering was if there was an easy way to implent multiple cores or somehow make the training run faster with synaptic. If you have any ideas please let me know! Thanks.

**Other Info:**

Training Size: 1,000 
Cost: MSE (I have tried CROSS_ENTROPY and BINARY. Pretty much no difference)

**Comment by Pummelchen** [comment]
For large networks like these you may consider to switch to Tensorflow and use a Nvidia 1080 upwards.

**Comment by bootsie123** [comment]
Alright, I'll look into that. Thanks! I also heard that Brain.js is a lot faster. Do you think it's worth switching?

**Comment by Pummelchen** [comment]
Well, I don't want to offend anyone, but for large data I think Tensorflow with Keras is currently the only way to go. You can easily use multiple GPUs with Tensorflow, too. Check the performance you get out of Tensorcores with the Titan V and how easy it is to use them. Synaptic is great for small networks and to develop concepts.

**Comment by bootsie123** [comment]
Ah. Thanks! I'll try it out then.
